scripts/newinstall.sh

```diff
@@ -142,11 +140,10 @@ if true; then
 			EOF
 
 			while true; do
-				read -p "Would you like to try continuiung without git? " yn
+				read -p "Would you like to try continuing without git? " yn
```

Comment:
There is a typo on line 138 above "you" -> "your" but I can't annotate that line.